Do not use enum_certificates, as it's not available on mingw.

From: Tom Rothamel <tom@rothamel.us>


---
 Lib/ssl.py |    7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/Lib/ssl.py b/Lib/ssl.py
index 30f4e59..21007cc 100644
--- a/Lib/ssl.py
+++ b/Lib/ssl.py
@@ -249,9 +249,6 @@ class _TLSMessageType(_IntEnum):
     CHANGE_CIPHER_SPEC = 0x0101
 
 
-if sys.platform == "win32":
-    from _ssl import enum_certificates, enum_crls
-
 from socket import socket, SOCK_STREAM, create_connection
 from socket import SOL_SOCKET, SO_TYPE
 import socket as _socket
@@ -569,9 +566,7 @@ class SSLContext(_SSLContext):
     def load_default_certs(self, purpose=Purpose.SERVER_AUTH):
         if not isinstance(purpose, _ASN1Object):
             raise TypeError(purpose)
-        if sys.platform == "win32":
-            for storename in self._windows_cert_stores:
-                self._load_windows_store_certs(storename, purpose)
+
         self.set_default_verify_paths()
 
     if hasattr(_SSLContext, 'minimum_version'):
